integrations/facebook-pixel/test/index.test.js

```diff
@@ -250,42 +259,11 @@ describe('Facebook Pixel', function() {
       });
 
       describe('segment ecommerce => FB product audiences', function() {
-        describe('Product List Viewed', function() {
```

Comment:
What's the reasoning behind deleting these tests?

Reply:
I might've migrated the code wrong here. I'll loop back around and fix.